src/utils/api-utils/ApiUtils.js

```diff
@@ -10,6 +11,15 @@ export const validateServiceErrors = async (
     });
   } catch (error) {
     console.error(error);
+    const allErrorsHandled = handleCustomErrors(error?.response?.data?.message);
```

Comment:
I’m a little bothered by not being able to understand what this function does and what it returns from its name. I’ve figured it out though and I do get it, and personally can’t think of a better name. I think this would particularly benefit from TypeScript to document what’s expected of this function (That’s obviously vastly out of scope for this ticket 😄 just thinking aloud).